contract/vm_callback.go

```diff
@@ -692,11 +692,11 @@ func LuaECVerify(L *LState, msg *C.char, sig *C.char, addr *C.char) C.int {
 	} else {
 		sign, err := btcec.ParseSignature(bSig, btcec.S256())
 		if err != nil {
-			luaPushStr(L, "[Contract.LuaEcVerify]Error signature parsing:"+err.Error())
+			luaPushStr(L, "[Contract.LuaEcVerify] error parsing signature: "+err.Error())
 			return -1
 		}
 		if pubKey == nil {
-			luaPushStr(L, "[Contract.LuaEcVerify]not supported")
+			luaPushStr(L, "[Contract.LuaEcVerify] pubKey cannot be empty")
```

Comment:
pubKey is not input. this error caused by invalid or unsupported address format that can not get pubkey

Reply:
is "error recovering pubKey" ok?

Reply:
ok